pkg/cmd/recoveryapiserver/create.go

```diff
@@ -0,0 +1,122 @@
+package recoveryapiserver
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"path"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/client-go/tools/watch"
+
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/recovery"
+)
+
+type CreateOptions struct {
+	Options
+
+	KubeApiserverImage string
+	Timeout            time.Duration
+	Wait               bool
+}
+
+func NewCreateCommand() *cobra.Command {
+	o := &CreateOptions{
+		Options:            NewDefaultOptions(),
+		KubeApiserverImage: "", // TODO: set the public pullspec
```

Comment:
what is the plan where to get this from?

Reply:
I think we can autodetect it from the current kubeapiserver if present or you can get that image from the release